versions.properties

```diff
@@ -9,85 +9,50 @@
 ####
 #### NOTE: Some versions are filtered by the rejectVersionIf predicate. See the settings.gradle.kts file.
 
-# Project versioning, run the following gradle command to update version numbers automatically:
-# ./gradlew bumpVersion --nextVersion=X.Y.Z
+# Project versioning
+#  NOTE: semantic version of the SDK lives in strings.xml
+#  run the following gradle command to update version numbers automatically:
+#  ./gradlew bumpVersion --nextVersion=X.Y.Z
 version.klaviyo.versionCode=20
 
-# Android versioning
+# Project Plugins
+plugin.android=8.2.1
+plugin.org.jetbrains.dokka=1.9.10
+plugin.org.jlleitschuh.gradle.ktlint=version.org.jlleitschuh.gradle..ktlint-gradle
 
+# Android versioning
+version.android.buildTools=34.0.0
+version.android.compileSdk=34
 version.android.minSdk=23
-
 version.android.targetSdk=34
 
-version.android.compileSdk=34
-
-version.android.buildTools=34.0.0
-
 # Project dependencies
-
-version.kotlinx.coroutines=1.9.0
-
-version.org.jetbrains.dokka..versioning-plugin=1.9.10
-##                                 # available=1.9.20
-
+version.androidx.activity=1.6.1
+version.androidx.appcompat=1.6.1
+version.androidx.compose.compiler=1.5.10
```

Comment:
This does NOT add compose as a dependency, only sets a version property for when the SDK + test app are used in composite mode.